src/GenericFactory.sol

```diff
@@ -79,7 +79,6 @@ contract GenericFactory is IGenericFactory, Owned {
             }
 
             lFreeMem += lSize;
-            // TODO: Do we need to pad to 32 bytes?
```

Comment:
this should be fine as it has been working without problems and from our understanding of constructor bytecode

Reply:
agreed, can resolve